### DISCUSSION TITLE
Slow startup

### DISCUSSION THREAD

**Discussion by Drallas** [alexjustesen/speedtest-tracker#705] [Q&A]
I'm using the MariaDB version and startup after deployment is slow (>10 minutes) and it's on a fast server! Anyone else seeing this too?

**Comment by alexjustesen** [comment]
This is related a file permission issue during startup regardless of DB and really shows itself on lower powered devices. #676 aims to fix this

**Reply by Drallas** [reply]
Tnx.. 

In my case, not a lower powered device, but a VM with Docker on:

<img width="1048" alt="Screenshot 2023-08-31 at 14 52 44" src="https://github.com/alexjustesen/speedtest-tracker/assets/24792888/d66a05ab-8351-461d-85c7-e1a698d1eab0">

```
root@pve01:~#
cat /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor
schedutil
schedutil
schedutil
```
